scripts/test-upgrade-path.sh

```diff
@@ -54,39 +73,35 @@ while :; do
     esac
 done
 
-if [ $# -lt 2 ]
-then
-    echo "Missing <from-version> or <to-version>"
-    usage
-    exit 1
-fi
-
-OLD_VERSION=$1
-NEW_VERSION=$2
-shift ; shift
 TESTS=$*
 
-NETWORK_NAME=migration-$OLD_VERSION-$NEW_VERSION
+ENV_PREFIX=${UPGRADE_ENV_PREFIX:-$(openssl rand -hex 8)}
 
-# Between docker-compose v1 and docker-compose v2, the delimiting
-# character for container names was changed from "-" to "_".
-if [[ "$(docker-compose --version)" =~ v2 ]]
-then
-    OLD_CONTAINER=$(gojira prefix -t $OLD_VERSION)-kong-1
-    NEW_CONTAINER=$(gojira prefix -t $NEW_VERSION)-kong-1
-else
-    OLD_CONTAINER=$(gojira prefix -t $OLD_VERSION)_kong_1
-    NEW_CONTAINER=$(gojira prefix -t $NEW_VERSION)_kong_1
-fi
+COMPOSE="docker compose -p $ENV_PREFIX -f scripts/upgrade-tests.yml"
+
+NETWORK_NAME=$ENV_PREFIX
+
+OLD_CONTAINER=$ENV_PREFIX-kong_old-1
+NEW_CONTAINER=$ENV_PREFIX-kong_new-1
+
+function prepare_container() {
+    docker exec $1 apt-get update
+    docker exec $1 apt-get install -y build-essential curl m4
+    docker exec $1 ln -sf /usr/lib/x86_64-linux-gnu/librt.so /usr/local/lib
```

Comment:
curious on why do we need this?

Reply:
I have not really found the reason why `librt.so` is not found in `/usr/lib/x86_64-linux-gnu/`, but I thought this was an acceptable workaround for a throwaway container.  I'll add a comment.